app/views/layouts/_navigation.html.erb

```diff
@@ -0,0 +1,29 @@
+<nav class="navbar navbar-light justify-content-between style-nabar">
+  <a class="navbar-brand" href="#">
+    <%= image_tag "study.png" %>
+    <span class="header-text">Improved Learning</span>
+  </a>
+  <form class="form-inline">
+    <div class="header-button-input">
+      <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
+      <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
+    </div>
+    <div class="form-group row">
+      <% if current_user  %>
+        <li><%= link_to "My Subjects", subjects_path, class: 'btn btn-primary black-btn' %></li>
```

Comment:
i would recommend to use translations everywhere: `t('subjects.my_subjects')` even if you have no near future plans to have multilingual site. It is total pain to hunt all untranslated places in bigger apps

Reply:
Oh, okey I would keep in mind that for next time